docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1391,6 +1391,27 @@ UDP
   TCP/UDP
     Not implemented ("-").
 
+%PATH(X:Y):Z%
+  HTTP
+    The value of the request path. The parameter X is used to specify should the output contains
+    query or not. The parameter Y is used to specify the position of the request path. Both X and Y
+    are optional.
+
+    The X parameter can be:
+
+    * ``WQ``: With Query. If the X is not presetn, ``WQ`` will be used.
```

Comment:
Maybe note what query is, to give people with less knowledge of http details more context